### PR TITLE
As send utm data at thank you page

### PR DESCRIPTION
**Pull request by artursmirnov**
Update for https://travisci.assembla.com/spaces/Web/tickets/281

### CODE REVIEW

File: app/services/utm.js [hunk 1]
```diff
@@ -41,28 +42,54 @@ export default Service.extend({
...
+    }, new QueryParamsHash());
+  },
+
+  capture(forceClear = false) {
+    if (this.hasParamsInUrl) {
+      UTM_FIELD_NAMES.forEach(field => {
+        const value = this.searchParams.get(field);
+        this.set(SERVICE_UTM_VARS[field], value);
+      });
+      if (forceClear) this.clearUrl();
```
Comment (sorta) [inline app/services/utm.js:70]: I don't see a `clearUrl` function, is it supposed to be `removeFromUrl`?

Comment (artursmirnov) [reply app/services/utm.js:70]: Ah, right, thanks for catching 👍 Forgot to rename.

**Comment by artursmirnov** [comment]
@emmanuel-ci, oh, seems like the last commit introduced this issue 😕 Thanks for the catch!

**Comment by artursmirnov** [comment]
Thanks everyone for reviews and feedback! 🎉

**Pull request merged**